### DISCUSSION TITLE
Match transparency level of app with other tray icon windows

### DISCUSSION THREAD

**Discussion by prakharb5** [File-New-Project/EarTrumpet#714] [Ideas]
Please match the transparency level of app with other tray icon of windows like of battery and network. It will make it look consistent.

**Comment by prakharb5** [comment]
![image](https://user-images.githubusercontent.com/51435948/91704755-a5a81080-eb99-11ea-84af-56829626ef34.png)

and 

![image](https://user-images.githubusercontent.com/51435948/91704815-b35d9600-eb99-11ea-8224-a190ad79db19.png)


(almost white background)

**Comment by riverar** [comment]
@Cyberdroid1 We don't have any insight into all the variables the Windows Shell team used for their flyout acrylic; getting the combination right is very difficult (and specific to each release of Windows 10). I'll take another look at it though, thanks.
